assets/queries/cloudFormation/iam_password_has_min_length/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "IAM_password_has_min_length",
+  "queryName": "IAM Password Has Min Length",
```

Comment:
The query name should reflect the vulnerability, not the fix. We should use something like the reverse of what it's now. Expand Min to Minimum